wiki/People/The_Team/pl.md

```diff
@@ -53,9 +54,18 @@ Grupy
 | ---- | ----------- |
 | [Moderatorzy globalni](/wiki/People/Global_Moderation_Team) | Sprawowanie porządku na forum i w czacie |
 | [Nominatorzy Beatmap](/wiki/People/Beatmap_Nomination_Group) | Osoby które zrobią wszystko, żeby twoja mapa była "zakwalifikowana" |
-| [Twórcy](http://osu.ppy.sh/g/11) | Tworzenie wspaniałej gry poprzez naprawianie bugów i dodawanie nowych opcji |
-| [osu! alumni](/wiki/People/osu!_Alumni) | Ludzie zapisani w historii osu! |
-| [Support Team Redux](/wiki/People/Support_Team) | Pomoc z prośbami nt. nowych rzeczy |
-| [osu!news](http://osu.ppy.sh/g/25) | Zapewnianie dobrej lektury od 2013 |
-| [Staff Turniejowy](http://osu.ppy.sh/g/26) | Ludzie odpowiadający za prowadzenie oficjalnych mistrzostw świata w osu! |
+| [Twórcy](http://osu.ppy.sh/g/11) | Sprawianie, że gra jest wspaniała poprzez naprawianie bugów i dodawanie nowych opcji |
+| [osu! alumni](/wiki/People/osu!_Alumni) | Ludzie, którzy zapisali się w historii osu! |
+| [Support Team Redux](/wiki/People/Support_Team) | Pomooc z prośbami i pytaniami  |
 | [Zespół zapewnienia jakości](/wiki/People/Quality_Assurance_Team) | Zapewniamy jakość beatmap, abyś mógł się dobrze bawić. |
```

Comment:
Zapewnianie jakości beatmap

Reply:
w angielskiej wersji zdanie jest napisane w pierwszej osobie, więc lepiej to zostawić.